hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1700,6 +1712,42 @@
       .</description>
   </property>
 
+  <property>
+    <name>ozone.om.ratis.snapshot.dir</name>
+    <value/>
+    <tag>OZONE, OM, STORAGE, MANAGEMENT, RATIS</tag>
+    <description>This directory is used for storing OM's snapshot
+      related files like the ratisSnapshotIndex and DB checkpoint from.
```

Comment:
Can we add a description, if this is not set the we will fall back to ozone.om.ratis.storage dir/ozone.om.metadirs.
And also add a description similar to other db dirs, fall back is not recommended for production.